index.js

```diff
@@ -0,0 +1,9 @@
+const camel_case_expected = (() => {
+    return 'Kya kar raha hai bhai tu'
+});
+
+
+const ThisIsHowItsDone = (() => {
+    // Logic that will definately fail.
+    camel_case_expected();
```

Comment:
naming is not correct
------------------------------------------------
ISSUE CODE:
JS001 - Variable naming convention should be correct